src/Admin/AmpThemes.php

```diff
@@ -60,6 +61,10 @@ public static function get_registration_action() {
 	 */
 	public static function is_needed() {
 
+		if ( ! Services::get( 'dependency_support' )->has_support() ) {
+			return false;
+		}
```

Comment:
As mentioned above in https://github.com/ampproject/amp-wp/pull/6773/files#r767010989, this code is causing an error upon activation:

![image](https://user-images.githubusercontent.com/134745/145646963-c631f9ed-a817-4186-9626-7f7cb82693ec.png)